### PR TITLE
Add empty www.conf to prevent issues when upgrading php-fpm modules

### PR DESCRIPTION
**Pull request by phavekes**

### CODE REVIEW

File: roles/app/tasks/main.yml [hunk 1]
```diff
@@ -134,12 +132,11 @@
...
 
-    - name: Remove known default distro files in /etc/opt/remi/php72/php-fpm.d/
-      file: path={{item}} state=absent
-      with_items:
-        - /etc/opt/remi/php72/php-fpm.d/www.conf
+    # Place empty www.conf to prevent this file being created on php-fpm OS-updates
+    - name: Add empty www.conf in /etc/opt/remi/php72/php-fpm.d/www.conf
+      copy: src='www.conf' dest='/etc/opt/remi/php72/php-fpm.d/www.conf'
       notify:
-        - restart php72-php-fpm
```
Comment (pmeulen) [inline roles/app/tasks/main.yml:142]: This looks like a cut&paste error.
Must restart php72-php-fpm after changing it's config

File: roles/app/tasks/main.yml [hunk 2]
```diff
@@ -186,8 +183,8 @@
   - restart nginx
 
 # This task can be removed once deployed on all nodes
-- name: Remove obsolete Nginx rsyslog config 
-  file: path=/etc/rsyslog.d/nginx.conf state=absent 
+- name: Remove obsolete Nginx rsyslog config
+  file: path=/etc/rsyslog.d/nginx.conf state=absent
   notify: restart rsyslog
```
Comment (pmeulen) [inline roles/app/tasks/main.yml:188]: This shouldn't be part of this PR / commit.

**Review by pmeulen — CHANGES_REQUESTED** [review]

File: roles/app/tasks/main.yml (hunk 1 above)
Comment (phavekes) [reply roles/app/tasks/main.yml:142]: Thank you for pointing this out.
